packages/react-composites/src/composites/ChatComposite/styles/Chat.styles.ts

```diff
@@ -54,8 +48,8 @@ export const chatHeaderContainerStyle = mergeStyles({
 });
 
 export const topicNameLabelStyle = mergeStyles({
-  fontSize: '1.1rem',
   lineHeight: '2.5rem',
+  fontSize: '1.1rem',
```

Comment:
Why reorder? If you are reordering, consider alphabetizing.

Reply:
unintentional change, was removed then put it back in - but slightly in the wrong place it seems

Reply:
removed change